p2panda-rs/src/schema/operation.rs

```diff
@@ -60,11 +59,11 @@ pub const OPERATION_SCHEMA: &str = r#"
 
     operation-value-relation = (
         type: "relation",
-        value: relation,
+        value: relation / pinned_relation,
     )
 
     operation-value-relation-list = (
         type: "relation_list",
-        value: [* relation],
+        value: [* relation] / [* pinned_relation],
```

Comment:
This is much neater / easier to understand!

Reply:
and it makes for neater CBOR!